<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>Attayeb Mohsen</title>
  <meta content="" name="description">
  <meta content="" name="keywords">


  <link href="assets/img/baseline_auto_stories_black_24dp.png" rel="icon">
  <!-- Favicons 
  <link href="assets/img/apple-touch-icon.png" rel="apple-touch-icon">
  -->
  <!-- Google Fonts -->
  <link
    href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Roboto:300,300i,400,400i,500,500i,600,600i,700,700i|Poppins:300,300i,400,400i,500,500i,600,600i,700,700i"
    rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
  <link href="assets/vendor/icofont/icofont.min.css" rel="stylesheet">
  <link href="assets/vendor/boxicons/css/boxicons.min.css" rel="stylesheet">
  <link href="assets/vendor/owl.carousel/assets/owl.carousel.min.css" rel="stylesheet">
  <link href="assets/vendor/venobox/venobox.css" rel="stylesheet">
  <link href="assets/vendor/aos/aos.css" rel="stylesheet">

  <!-- css.gg -->
  <link href='https://css.gg/css' rel='stylesheet'>

  <!-- UNPKG -->
  <link href='https://unpkg.com/css.gg/icons/all.css' rel='stylesheet'>

  <!-- JSDelivr -->
  <link href='https://cdn.jsdelivr.net/npm/css.gg/icons/all.css' rel='stylesheet'>

  <!-- fontawesome -->
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
    integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w=="
    crossorigin="anonymous" />

  <!-- academicons 1.9.1 -->
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.1/css/academicons.min.css"
    integrity="sha512-b1ASx0WHgVFL5ZQhTgiPWX+68KjS38Jk87jg7pe+qC7q9YkEtFq0z7xCglv7qGIs/68d3mAp+StfC8WKC5SSAg=="
    crossorigin="anonymous" />

  <!-- bootstrap icons -->
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.4.1/font/bootstrap-icons.css">

  <!-- Template Main CSS File -->
  <link href="assets/css/style.css" rel="stylesheet">


  <!-- =======================================================
  * Template Name: BizLand - v1.2.1
  * Template URL: https://bootstrapmade.com/bizland-bootstrap-business-template/
  * Author: BootstrapMade.com
  * License: https://bootstrapmade.com/license/
  ======================================================== -->
</head>

<body>

  <!-- ======= Top Bar ======= -->
  <div id="topbar" class="d-none d-lg-flex align-items-center fixed-top">
    <div class="container d-flex">
      <div class="contact-info mr-auto" , style="font-size: 1.5rem; vertical-align: middle;">
        <i class="icofont-envelope"></i> <a href="mailto:attayeb@nibiohn.go.jp">attayeb@nibiohn.go.jp</a>
      </div>
      <div class="social-links" style="font-size: 2rem; vertical-align: bottom;">
        <a href="https://www.linkedin.com/in/attayebmohsen/" class="linkedin"><i class="icofont-linkedin"></i></i></a>
        <a href="https://www.researchgate.net/profile/Attayeb-Mohsen" class="researchgate"><i
            class="fab fa-researchgate"></i></i></a>
        <a href="https://scholar.google.com/citations?user=SPmoxAoAAAAJ&hl=en" class="googlescholar"><i
            class="ai ai-google-scholar-square"></i></i></a>
        <a href="https://orcid.org/0000-0003-0690-8012" class="googlescholar"><i class="fab fa-orcid"></i></i></a>

      </div>
    </div>
  </div>

  <!-- ======= Header ======= -->
  <header id="header" class="fixed-top">
    <div class="container d-flex align-items-center">

      <h1 class="logo mr-auto"><a href="index.html">Home<span>.</span></a></h1>
      <!-- Uncomment below if you prefer to use an image logo -->
      <!-- <a href="index.html" class="logo mr-auto"><img src="assets/img/logo.png" alt=""></a>-->

      <nav class="nav-menu d-none d-lg-block">
        <ul>
          <li class="active"><a href="index.html">Home</a></li>
          <li><a href="#about">About</a></li>
          <li><a href="#skills">Skills</a></li>
          <li><a href="#publications">Publications</a></li>
          <li><a href="#contact">Contact</a></li>
          <!-- <li><a href="#team">Team</a></li> -->
          <!-- <li class="drop-down"><a href="">Drop Down</a>
            <ul>
              <li><a href="#">Drop Down 1</a></li>
              <li class="drop-down"><a href="#">Deep Drop Down</a>
                <ul>
                  <li><a href="#">Deep Drop Down 1</a></li>
                  <li><a href="#">Deep Drop Down 2</a></li>
                  <li><a href="#">Deep Drop Down 3</a></li>
                  <li><a href="#">Deep Drop Down 4</a></li>
                  <li><a href="#">Deep Drop Down 5</a></li>
                </ul>
              </li>
              <li><a href="#">Drop Down 2</a></li>
              <li><a href="#">Drop Down 3</a></li>
              <li><a href="#">Drop Down 4</a></li>
            </ul>
          </li> -->


        </ul>
      </nav><!-- .nav-menu -->

    </div>
  </header><!-- End Header -->

  <!-- ======= Hero Section ======= -->
  <section id="hero" class="d-flex align-items-center">
    <div class="container" data-aos="zoom-out" data-aos-delay="100">
      <h3>Interactive resume</h3>
      <h1><span>Attayeb Mohsen</span>
        </h3>
        <h2>AI center for Health and Biomedical <br>Research (ArCHER) (NIBIOHN)</h2>
        <!-- 
      <div class="d-flex">
        <a href="#about" class="btn-get-started scrollto">Get Started</a>
        <a href="https://www.youtube.com/watch?v=jDDaplaOz7Q" class="venobox btn-watch-video" data-vbtype="video" data-autoplay="true"> Watch Video <i class="icofont-play-alt-2"></i></a>
      </div>
      -->
    </div>
  </section><!-- End Hero -->

  <main id="main">

    <!-- ======= Featured Services Section ======= -->
    <section id="featured-services" class="featured-services">
      <div class="container p-5" data-aos="fade-up">

        <!-- <div class="row"> -->
        <!-- 
            <div class="col-md-6 col-lg-6 d-flex align-items-stretch mb-5 mb-lg-0">
            -->
        <div class="icon-box p-3 mb-3" data-aos="fade-up" data-aos-delay="100">
          <!-- <div class="icon"><i class="fa fa-book fa-7x"></i></div> -->
          <h4 class="title description"> Who am I </h4>
          <p class="description">Physician and Computational Biologist with experience in Bioinformatics, AI, and Big
            data analysis, including NGS data.</p>
        </div>
        <!--
          </div>
-->

        <!-- <div class="col-md-6 col-lg-6 d-flex align-items-stretch mb-5 mb-lg-0"></div> -->
        <div class="icon-box p-3 mb-3" data-aos="fade-up" data-aos-delay="200">
          <!-- <div class="icon"><i class="fas fa-baby fa-sm"></i></div> -->
          <h4 class="title description">Introduction</h4>
          <p class="description mt-2">
          <ul class="description list-group-flush">
            <li>Passionate about Artificial intelligence in medicine and health.</li>
            <li>Committed to collaborate and exchange experiences and ideas with peer researchers. </li>
            <li>Strive for self-development by learning and acquiring new skills and exploring new science and research
              areas. </li>
            <li> Active in journal articles writing and conference participation.</li>
            <li>Enthusiastic in Health-related problem solving utilizing programming and computational methods.</li>
          </ul>
          </p>
        </div>
        <!-- </div> -->



        <!-- </div> -->

      </div>
    </section><!-- End Featured Services Section -->

    <!-- ======= About Section ======= -->
    <section id="about" class="about section-bg">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>About</h2>
          <h3>Find Out More <span>About Me</span></h3>
          <p> . . . </p>
        </div>

        <div class="row">
          <!-- <div class="col-lg-6" data-aos="zoom-out" data-aos-delay="100">
            <img src="assets/img/about.jpg" class="img-fluid" alt="">
          </div> -->

          <div class="col-lg-6 pt-4 pt-lg-0 content d-flex flex-column justify-content-center" data-aos="fade-up"
            data-aos-delay="100">
            <h3>Eduction History</h3>
            <ul>
              <li>
                <i class="bx"></i>
                <div>
                  <h5>Tohoku University Graduate School of Medicine</h5>
                  <p>
                    (2014) Ph. D. Medical Sciences; in Pharmacology department.
                  </p>
                </div>
              </li>
              <li>
                <i class="bx"></i>
                <div>
                  <h5>University of Pretoria</h5>
                  <p>
                    (2009) Post-graduate Diploma in Occupational Medicine and Health.
                  </p>
                </div>
              </li>
              <li>
                <i class="bx"></i>
                <div>
                  <h5>University of Benghazi</h5>
                  <p>
                    (2006) MB. ChB. Bachelor of Medicine and Surgery.
                  </p>
                </div>
              </li>
            </ul>
          </div>

          <div class="col-lg-6 pt-4 pt-lg-0 content d-flex flex-column justify-content-center" data-aos="fade-up"
            data-aos-delay="100">
            <h3>Employment History</h3>
            <ul>
              <li>
                <i class="bx"></i>
                <div>
                  <h5>National Institutes of Biomedical Innovation, Health and Nutrition</h5>
                  <p>
                    (2015 ~) Post-doctoral researcher.
                  </p>
                </div>
              </li>
              <li>
                <i class="bx"></i>
                <div>
                  <h5>Tohoku University, Cyclotron and Radio-Isotope Center</h5>
                  <p>
                    (2014) Post-doctoral researcher.
                  </p>
                </div>
              </li>
              <li>
                <i class="bx"></i>
                <div>
                  <h5>University of Benghazi</h5>
                  <p>
                    (2006) MB. ChB. Bachelor of Medicine and Surgery.
                  </p>
                </div>
              </li>
            </ul>
          </div>
        </div>

      </div>
    </section><!-- End About Section -->


    <!-- Detailed Education -->
    <section id="detailed_education" class="faq section-bg">
      <div class="container" data-aos="fade-up">
        <div class="p-5">
          <div class="section-title">
            <h2>Education and Employment in details</h2>
            <h3>Education and Employment in <span>details</span></h3>
            <p>Comment here</p>
          </div>

          <ul class="faq-list" data-aos="fade-up" data-aos-delay="100">
            <!-- Education -->
            <h4>Education</h4>
            <li>
              <a data-toggle="collapse" href="#edu1" class="collapsed">
                PhD in Medical Sciences
                <i class="icofont-simple-up"></i></a>
              <div id="edu1" class="collapse" data-parent=".faq-list">

                <p class="p-3">
                  <img src="assets/img/tohokuuniversitylogo.gif" alt="">
                <ul>
                  <li>
                    Tohoku University
                  </li>
                  <li>
                    <a href="https://www.med.tohoku.ac.jp/english/about/laboratory/011.html"
                      class="fas fa-facebook">Department of Pharmacology, Graduate school of Medicine, Sendai, Japan
                    </a>
                  </li>
                  <li>
                    Thesis: The role of histaminergic neurotransmission in locomotor activity and anxiety-like-behaviors
                    in mice.
                  </li>
                </ul>
                </p>
              </div>
            </li>
            <li>
              <a data-toggle="collapse" href="#edu2" class="collapsed">
                Post Graduate Diploma in Occupation Medicine and Health
                <i class="icofont-simple-up"></i></a>
              <div id="edu2" class="collapse" data-parent=".faq-list">
                <p class="p-3">

                <ul>
                  <li>
                    University of Pretoria.
                  </li>
                  <li>School of Health Systems and Public Health, Pretoria, South Africa</li>
                </ul>

                </p>
              </div>
            </li>

            <li>
              <a data-toggle="collapse" href="#edu3" class="collapsed">
                Bachelor of Medicine and Surgery (MB ChB)
                <i class="icofont-simple-up"></i></a>
              <div id="edu3" class="collapse" data-parent=".faq-list">
                <p class="p-3">

                <ul>
                  <li>
                    University of Benghazi.
                  </li>
                  <li>School of Medicine, Benghazi, Libya</li>
                </ul>

                </p>
              </div>
            </li>

            <!-- Employment -->

            <h4>Employment</h4>
            <li>
              <a data-toggle="collapse" href="#emp1" class="collapsed">
                Post-doctoral Researcher<br>
                National Institutes of Biomedical Innovation, Health and Nutrition<br>
                Artificial Intelligence Center for Health and Biomedical Research (ArCHER)<br>
                Laboratory of Bioinformatics
                <i class="icofont-simple-up"></i></a>
              <div id="emp1" class="collapse" data-parent=".faq-list">
                <p class="p-3">

                <ul>
                  <li>
                    In the theme of drug discovery, I created prediction models for adverse drug
                    events. This project utilized data from different sources (ADR (Adverse Drug Re-
                    actions) database, Microarray data), it uses machine learning for feature
                    selection and outcome prediction.
                    (Won the poster excellent award in Japanese Chemo-bioinformatics meeting Tokyo,
                    2016). In this project
                    the target genes of these adverse events were identified, and prediction models
                    are created to be used for
                    new drug development settings.
                  </li>
                  <li>
                    Worked on natural language processing (NLP) with collaborators to
                    analyze the similarity of terms used in medical literature using Word2Vec method
                    (Results partially com-
                    municated as a poster presentation in CBI 2018).
                  </li>
                  <li>
                    Built a deep learning model to remove the noise from MCG (MagnetoCardioGram)
                    traces
                    in collaboration with researchers from multiple international universities.
                  </li>
                  <li>
                    Set up the Microbiome data analysis pipeline in our laboratory for both 16S data
                    (using
                    QIIME 1 and QIIME 2) and Shot-gun data (abundance estimation and functional
                    annotation).

                  </li>
                  <li>
                    Automated and optimized QIIME1 pipe line by developing (Auto-q) script.
                  </li>
                  <li>
                    Trained a research group from other university on the automation and
                    optimization of Microbiome data
                    analysis using QIIME 1 pipeline and auto-q script.

                  </li>
                  <li>
                    Commenced and organized a technical training sessions in the laboratory under
                    the name of Interactive
                    Workshop. Their aim is to share the experience among team members and invite
                    experts from outside the
                    laboratory.

                  </li>
                  <li>
                    Studied the spread of COVID-19 infection using stochastic simulation with
                    involvement of
                    network approach.
                  </li>
                  <li>
                    Deep learning and conventional machine learning.
                  </li>
                  <li>
                    Bayesian inference and network analysis.
                  </li>
                </ul>
                </p>
              </div>
            </li>
            <li>
              <a data-toggle="collapse" href="#emp2" class="collapsed">
                Post-doctoral Researcher<br>
                Tohoku University Cyclotron and Radio-isotope center
                <i class="icofont-simple-up"></i></a>
              <div id="emp2" class="collapse" data-parent=".faq-list">
                <p class="p-3">

                <ul>
                  <li>
                    NIRS (Near InfraRed Spectrophotometry)
                  </li>
                  <li>
                    PET (Positron emission tomography)
                  </li>
                  <li>
                    Participated in a project to study the effect of Antihistamines on the brain blood supply using PET
                    imaging
                    and NIRS, Developed a script to record the response of the experiment subjects and analyze their
                    cognitive
                    abilities.
                  </li>
                </ul>

                </p>
              </div>
            </li>

            <li>
              <a data-toggle="collapse" href="#emp3" class="collapsed">
                General Practitioner
                <i class="icofont-simple-up"></i></a>
              <div id="emp3" class="collapse" data-parent=".faq-list">
                <p class="p-3">

                <ul>
                  <li>
                    Follow up of the in-patients in Medical ward in Tripoli Central Hospital.
                  </li>
                  <li>
                    Deliver emergency care in Tripoli Central Hospital Medical department, and National Heart Center
                    Cardiology Emergency Room.
                  </li>
                </ul>

                </p>
              </div>
            </li>
          </ul>
        </div>
      </div>
    </section>





    <!-- ======= Skills Section ======= -->



    <section id="skills" class="services">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>Skills</h2>
          <h3>Check my <span>Skills</span></h3>
          <p>My skills is here</p>
        </div>

        <div class="row">
          <div class="col-lg-4 col-md-6 d-flex align-items-stretch" data-aos="zoom-in" data-aos-delay="100">
            <div class="icon-box">
              <div class="icon"><i class="fa fa-laptop"></i></div>
              <h4>Programming skills</h4>
              <p>Python, R (Bioconductor), SQL, Sparql, Big-data, Cloud computing, Simulation, Bayesian modeling and
                inference, Automation and optimization, Simulation</p>
            </div>
          </div>

          <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4 mt-md-0" data-aos="zoom-in"
            data-aos-delay="200">
            <div class="icon-box">
              <div class="icon"><i class="fas fa-brain"></i></div>
              <h4>Neuroscience</h4>
              <p>Neuroparmacology of Histamine in Anxiety and sleep deprivation</p>
            </div>
          </div>

          <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4 mt-lg-0" data-aos="zoom-in"
            data-aos-delay="300">
            <div class="icon-box">
              <div class="icon"><i class="bx bx-dna"></i></div>
              <h4>Genomics</h4>
              <p>Micro-Array data analysis, NGS, Toxicogenomics, Microbiomics</p>
            </div>
          </div>

          <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4" data-aos="zoom-in" data-aos-delay="100">
            <div class="icon-box">
              <div class="icon"><i class="icofont-microscope"></i></div>
              <h4>Wet lab experience</h4>
              <p>PCR, HPLC, Immunohistochemisty, Mouse behaviour, Cell culture techniques</p>
            </div>
          </div>

          <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4" data-aos="zoom-in" data-aos-delay="200">
            <div class="icon-box">
              <div class="icon"><i class="bis bi-gear"></i></div>
              <h4>Machine learning</h4>
              <p>Deep learning(Keras, Tensorflow), Machine learning(Random forest, SVM, XG-Boost)</p>
            </div>
          </div>

          <div class="col-lg-4 col-md-6 d-flex align-items-stretch mt-4" data-aos="zoom-in" data-aos-delay="300">
            <div class="icon-box">
              <div class="icon"><i class="icofont-molecule"></i></div>
              <h4>Systems Biology</h4>
              <p>Pathway analysis, Gene enrichment analysis, Coexpression analysis</p>
            </div>
          </div>

        </div>

      </div>
    </section>


    <!-- ======= Counts Section ======= -->
    <section id="counts" class="counts">
      <div class="container" data-aos="fade-up">

        <div class="row">

          <div class="col-lg-6 col-md-6">
            <div class="count-box">
              <i class="icofont-paper"></i>
              <span data-toggle="counter-up">19</span>
              <p>Published papers</p>
            </div>
          </div>

          <div class="col-lg-6 col-md-6 mt-5 mt-md-0">
            <div class="count-box">
              <i class="icofont-history"></i>
              <span data-toggle="counter-up">326</span>
              <p>Citations</p>
            </div>
          </div>
        </div>
      </div>
    </section>

    <!-- Publications -->
    <section id="publications" class="faq section-bg">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>Publications</h2>
          <h3>Selected publications <span>list</span></h3>
          <p>Check our list of publication</p>
        </div>

        <ul class="faq-list" data-aos="fade-up" data-aos-delay="100">
          
          <h4>2022</h4>
          <li>
            <a data-toggle="collapse" href="#pub2022_1" class="collapsed">
              Co-expression analysis to identify key modules and hub genes associated with COVID-19 in platelets
              <i
                class="icofont-simple-up"></i></a>
            <div id="pub2022_1" class="collapse" data-parent=".faq-list">
              <p class="p-3">
              <h5>Authors</h5>
              <p>
                Ahmed B Alarabi, <strong>Attayeb Mohsen</strong>, Kenji Mizuguchi, Fatima Z Alshbool, Fadi T Khasawneh
              </p>
              <h5>Journal</h5>
              <p>
                BMC Med Genomics 15, 83 (2022).
              </p>

              <h5>Abstract</h5>
              <p class="pr-3 pl-3">
                Corona virus disease 2019 (COVID-19) increases the risk of cardiovascular occlusive/thrombotic events and is linked to poor outcomes. The underlying pathophysiological processes are complex, and remain poorly understood. To this end, platelets play important roles in regulating the cardiovascular system, including via contributions to coagulation and inflammation. There is ample evidence that circulating platelets are activated in COVID-19 patients, which is a primary driver of the observed thrombotic outcome. However, the comprehensive molecular basis of platelet activation in COVID-19 disease remains elusive, which warrants more investigation. Hence, we employed gene co-expression network analysis combined with pathways enrichment analysis to further investigate the aforementioned issues. Our study revealed three important gene clusters/modules that were closely related to COVID-19. These cluster of genes successfully identify COVID-19 cases, relative to healthy in a separate validation data set using machine learning, thereby validating our findings. Furthermore, enrichment analysis showed that these three modules were mostly related to platelet metabolism, protein translation, mitochondrial activity, and oxidative phosphorylation, as well as regulation of megakaryocyte differentiation, and apoptosis, suggesting a hyperactivation status of platelets in COVID-19. We identified the three hub genes from each of three key modules according to their intramodular connectivity value ranking, namely: COPE, CDC37, CAPNS1, AURKAIP1, LAMTOR2, GABARAP MT-ND1, MT-ND5, and MTRNR2L12. Collectively, our results offer a new and interesting insight into platelet involvement in COVID-19 disease at the molecular level, which might aid in defining new targets for treatment of COVID-19–induced thrombosis.
              </p>
              <a href="https://doi.org/10.1186/s12920-022-01222-y"
                target="_blank"><small>https://doi.org/10.1186/s12920-022-01222-y</small></a>
              </p>
            </div>
          </li>
          <h4>2021</h4>
          <li>
            <a data-toggle="collapse" href="#pub2021_3" class="collapsed">
              Deep Learning Prediction of Adverse Drug Reactions in Drug Discovery Using Open TG–GATEs and FAERS Databases
              <i
                class="icofont-simple-up"></i></a>
            <div id="pub2021_3" class="collapse" data-parent=".faq-list">
              <p class="p-3">
              <h5>Authors</h5>
              <p>
                <strong>Attayeb Mohsen</strong>, Lokesh P. Tripathi, Kenji Mizuguchi
              </p>
              <h5>Journal</h5>
              <p>
                Front. Drug. Discov., 27 October 2021
              </p>

              <h5>Abstract</h5>
              <p class="pr-3 pl-3">
                Machine learning techniques are being increasingly used in the analysis of clinical and omics data. This increase is primarily due to the advancements in Artificial intelligence (AI) and the build-up of health-related big data. In this paper we have aimed at estimating the likelihood of adverse drug reactions or events (ADRs) in the course of drug discovery using various machine learning methods. We have also described a novel machine learning-based framework for predicting the likelihood of ADRs. Our framework combines two distinct datasets, drug-induced gene expression profiles from Open TG–GATEs (Toxicogenomics Project–Genomics Assisted Toxicity Evaluation Systems) and ADR occurrence information from FAERS (FDA [Food and Drug Administration] Adverse Events Reporting System) database, and can be applied to many different ADRs. It incorporates data filtering and cleaning as well as feature selection and hyperparameters fine tuning. Using this framework with Deep Neural Networks (DNN), we built a total of 14 predictive models with a mean validation accuracy of 89.4%, indicating that our approach successfully and consistently predicted ADRs for a wide range of drugs. As case studies, we have investigated the performances of our prediction models in the context of Duodenal ulcer and Hepatitis fulminant, highlighting mechanistic insights into those ADRs. We have generated predictive models to help to assess the likelihood of ADRs in testing novel pharmaceutical compounds. We believe that our findings offer a promising approach for ADR prediction and will be useful for researchers in drug discovery.
              </p>
              <a href="https://doi.org/10.3389/fddsv.2021.768792"
                target="_blank"><small>https://doi.org/10.3389/fddsv.2021.768792</small></a>
              </p>
            </div>
          </li>
          <li>
            <a data-toggle="collapse" href="#pub2021_2" class="collapsed">Comprehensive analysis of gut microbiota of a
              healthy population and covariates affecting microbial variation in two large Japanese cohorts <i
                class="icofont-simple-up"></i></a>
            <div id="pub2021_2" class="collapse" data-parent=".faq-list">
              <p class="p-3">
              <h5>Authors</h5>
              <p>
                Jonguk Park, Kumiko Kato, Haruka Murakami, Koji Hosomi, Kumpei Tanisawa, Takashi Nakagata, Harumi Ohno,
                Kana Konishi, Hitoshi Kawashima, Yi-An Chen, <strong>Attayeb Mohsen</strong>, Jin-zhong Xiao, Toshitaka
                Odamaki, Jun Kunisawa, Kenji Mizuguchi, Motohiko Miyachi
              </p>
              <h5>Journal</h5>
              <p>
                BMC microbiology 21, Article number: 151 (2021).
              </p>

              <h5>Abstract</h5>
              <p class="pr-3 pl-3">
                <h6>Background</h6>
                <p>Inter-individual variations in gut microbiota composition are observed even among healthy populations.
                The gut microbiota may exhibit a unique composition depending on the country of origin and race of
                individuals. To comprehensively understand the link between healthy gut microbiota and host state, it is
                beneficial to conduct large-scale cohort studies. The aim of the present study was to elucidate the
                integrated and non-redundant factors associated with gut microbiota composition within the Japanese
                population by 16S rRNA sequencing of fecal samples and questionnaire-based covariate analysis.</p>

                <h6>Results</h6>
                <p>A total of 1596 healthy Japanese individuals participated in this study via two independent cohorts,
                NIBIOHN cohort (n = 954) and MORINAGA cohort (n = 642). Gut microbiota composition was described and the
                interaction of these microorganisms with metadata parameters such as anthropometric measurements, bowel
                habits, medical history, and lifestyle were obtained. Thirteen genera, including Alistipes,
                Anaerostipes, Bacteroides, Bifidobacterium, Blautia, Eubacterium halli group, Faecalibacterium,
                Fusicatenibacter, Lachnoclostridium, Parabacteroides, Prevotella_9, Roseburia, and Subdoligranulum were
                predominant among the two cohorts. On the basis of univariate analysis for overall microbiome variation,
                18 matching variables exhibited significant association in both cohorts. A stepwise redundancy analysis
                revealed that there were four common covariates, Bristol Stool Scale (BSS) scores, gender, age, and
                defecation frequency, displaying non-redundant association with gut microbial variance.</p>

                <h6>Conclusions</h6>
                <p>
                  We conducted a comprehensive analysis of gut microbiota in healthy Japanese individuals, based on two
                  independent cohorts, and obtained reliable evidence that questionnaire-based covariates such as
                  frequency of bowel movement and specific dietary habit affects the microbial composition of the gut. To
                  our knowledge, this was the first study to investigate integrated and non-redundant factors associated
                  with gut microbiota among Japanese populations.
                </p>
              </p>
              <a href="https://doi.org/10.1186/s12866-021-02215-0"
                target="_blank"><small>https://doi.org/10.1186/s12866-021-02215-0</small></a>
              </p>
            </div>
          </li>
          <li>
            <a data-toggle="collapse" href="#pub1" class="collapsed">Distinct Age-Specific miRegulome Profiling of
              Isolated Small and Large Intestinal Epithelial Cells in Mice <i class="icofont-simple-up"></i></a>
            <div id="pub1" class="collapse" data-parent=".faq-list">
              <p class="p-3">
              <h5>Authors</h5>
              <p>
                Juneyoung Lee, <strong>Attayeb Mohsen</strong>, Anik Banerjee, Louise D. McCullough, Kenji Mizuguchi,
                Motomu Shimaoka, Hiroshi Kiyono, Eun J. Park.
              </p>
              <h5>Journal</h5>
              <p>
                Int. J. Mol. Sci. 22, no. 7: 3544. (2021)
              </p>

              <h5>Abstract</h5>
              <p class="pr-3 pl-3">
                The intestinal epithelium serves as a dynamic barrier to protect the host tissue from exposure to a
                myriad of inflammatory stimuli in the luminal environment. Intestinal epithelial cells (IECs) encompass
                differentiated and specialized cell types that are equipped with regulatory genes, which allow for
                sensing of the luminal environment. Potential inflammatory cues can instruct IECs to undergo a diverse
                set of phenotypic alterations. Aging is a primary risk factor for a variety of diseases; it is now
                well-documented that aging itself reduces the barrier function and turnover of the intestinal
                epithelium, resulting in pathogen translocation and immune priming with increased systemic inflammation.
                In this study, we aimed to provide an effective epigenetic and regulatory outlook that examines
                age-associated alterations in the intestines through the profiling of microRNAs (miRNAs) on isolated
                mouse IECs. Our microarray analysis revealed that with aging, there is dysregulation of distinct
                clusters of miRNAs that was present to a greater degree in small IECs (22 miRNAs) compared to large IECs
                (three miRNAs). Further, miRNA–mRNA interaction network and pathway analyses indicated that aging
                differentially regulates key pathways between small IECs (e.g., toll-like receptor-related cascades) and
                large IECs (e.g., cell cycle, Notch signaling and small ubiquitin-related modifier pathway). Taken
                together, current findings suggest novel gene regulation pathways by epithelial miRNAs in aging within
                the gastrointestinal tissues
              </p>
              <a href="https://doi.org/10.3390/ijms22073544"
                target="_blank"><small>https://doi.org/10.3390/ijms22073544</small></a>
              </p>
            </div>
          </li>
          <h4>2020</h4>
          <li>
            <a data-toggle="collapse" href="#pub2020_1" class="collapsed">
              BioHackathon 2015: Semantics of data for life sciences and reproducible research
              <i class="icofont-simple-up"></i></a>
            <div id="pub2020_1" class="collapse" data-parent=".faq-list">
              <p class="p-3">
              <h5>Authors</h5>
              <p>
                Rutger A Vos, Toshiaki Katayama, Hiroyuki Mishima, Shin Kawano, Shuichi Kawashima, Jin-Dong Kim, Yuki Moriya, Toshiaki Tokimatsu, Atsuko Yamaguchi, Yasunori Yamamoto, Hongyan Wu, Peter Amstutz, Erick Antezana, Nobuyuki P Aoki, Kazuharu Arakawa, Jerven T Bolleman, Evan Bolton, Raoul JP Bonnal, Hidemasa Bono, Kees Burger, Hirokazu Chiba, Kevin B Cohen, Eric W Deutsch, Jesualdo T Fernández-Breis, Gang Fu, Takatomo Fujisawa, Atsushi Fukushima, Alexander García, Naohisa Goto, Tudor Groza, Colin Hercus, Robert Hoehndorf, Kotone Itaya, Nick Juty, Takeshi Kawashima, Jee-Hyub Kim, Akira R Kinjo, Masaaki Kotera, Kouji Kozaki, Sadahiro Kumagai, Tatsuya Kushida, Thomas Lütteke, Masaaki Matsubara, Joe Miyamoto, <strong>Attayeb Mohsen</strong>, Hiroshi Mori, Yuki Naito, Takeru Nakazato, Jeremy Nguyen-Xuan, Kozo Nishida, Naoki Nishida, Hiroyo Nishide, Soichi Ogishima, Tazro Ohta, Shujiro Okuda, Benedict Paten, Jean-Luc Perret, Philip Prathipati, Pjotr Prins, Núria Queralt-Rosinach, Daisuke Shinmachi, Shinya Suzuki, Tsuyosi Tabata, Terue Takatsuki, Kieron Taylor, Mark Thompson, Ikuo Uchiyama, Bruno Vieira, Chih-Hsuan Wei, Mark Wilkinson, Issaku Yamada, Ryota Yamanaka, Kazutoshi Yoshitake, Akiyasu C Yoshizawa, Michel Dumontier, Kenjiro Kosaki, Toshihisa Takagi
              </p>
              <h5>Journal</h5>
              <p>
                F1000Research, 2020;9:136.
              </p>
              <h5>Abstract</h5>
              <p class="pr-3 pl-3">
                We report on the activities of the 2015 edition of the BioHackathon, an annual event that brings together researchers and developers from around the world to develop tools and technologies that promote the reusability of biological data. We discuss issues surrounding the representation, publication, integration, mining and reuse of biological data and metadata across a wide range of biomedical data types of relevance for the life sciences, including chemistry, genotypes and phenotypes, orthology and phylogeny, proteomics, genomics, glycomics, and metabolomics. We describe our progress to address ongoing challenges to the reusability and reproducibility of research results, and identify outstanding issues that continue to impede the progress of bioinformatics research. We share our perspective on the state of the art, continued challenges, and goals for future research and development for the life sciences Semantic Web.
              </p>
              <a href="https://dx.doi.org/10.12688%2Ff1000research.18236.1"
                target="_blank"><small>https://dx.doi.org/10.12688%2Ff1000research.18236.1</small></a>
              </p>
            </div>
          </li>
          <li>
            <a data-toggle="collapse" href="#pub2" class="collapsed">MANTA, an integrative database and analysis
              platform that relates microbiome and phenotypic data <i class="icofont-simple-up"></i></a>
            <div id="pub2" class="collapse" data-parent=".faq-list">
              <p class="p-3">
              <h5>Authors</h5>
              <p>
                Yi-An Chen, Jonguk Park, Yayoi Natsume-Kitatani, Hitoshi Kawashima, <strong>Attayeb Mohsen</strong>,
                Koji Hosomi, Kumpei Tanisawa, Harumi Ohno, Kana Konishi, Haruka Murakami, Motohiko Miyachi, Jun
                Kunisawa, Kenji Mizuguchi
              </p>
              <h5>Journal</h5>
              <p>
                Plos one 2020, 15:12:e0243609
              </p>
              <h5>Abstract</h5>
              <p class="pr-3 pl-3">
                With an ever-increasing interest in understanding the relationships between the microbiota and the host,
                more tools to map, analyze and interpret these relationships have been developed. Most of these tools,
                however, focus on taxonomic profiling and comparative analysis among groups, with very few analytical
                tools designed to correlate microbiota and the host phenotypic data. We have developed a software
                program for creating a web-based integrative database and analysis platform called MANTA (Microbiota And
                pheNoType correlation Analysis platform). In addition to storing the data, MANTA is equipped with an
                intuitive user interface that can be used to correlate the microbial composition with phenotypic
                parameters. Using a case study, we demonstrated that MANTA was able to quickly identify the significant
                correlations between microbial abundances and phenotypes that are supported by previous studies.
                Moreover, MANTA enabled the users to quick access locally stored data that can help interpret
                microbiota-phenotype relations. MANTA is available at https://mizuguchilab.org/manta/ for download and
                the source code can be found at https://github.com/chenyian-nibio/manta.
              </p>
              <a href="https://doi.org/10.1371/journal.pone.0243609"
                target="_blank"><small>https://doi.org/10.1371/journal.pone.0243609</small></a>
              </p>
            </div>
          </li>
          <li>
            <a data-toggle="collapse" href="#pub3" class="collapsed">AI aided noise processing of spintronic based iot
              sensor for magnetocardiography application<i class="icofont-simple-up"></i></a>
            <div id="pub3" class="collapse" data-parent=".faq-list">
              <p class="p-3">
              <h5>Authors</h5>
              <p>
                <strong>Attayeb Mohsen</strong>, Muftah Al-Mahdawi, Mostafa M Fouda, Mikihiko Oogane, Yasuo Ando, Zubair
                Md Fadlullah
              </p>
              <h5>Journal</h5>
              <p>
                ICC 2020 - 2020 IEEE International Conference on Communications (ICC), Dublin, Ireland,
                2020, pp. 1-6
              </p>
              <h5>Abstract</h5>
              <p class="pr-3 pl-3">
                As we are about to embark upon the highly hyped “Society 5.0”, powered by the Internet of Things (IoT),
                traditional ways to monitor human heart signals for tracking cardio-vascular conditions are challenging,
                particularly in remote healthcare settings. On the merits of low power consumption, portability, and
                non-intrusiveness, there are no suitable IoT solutions that can provide information comparable to the
                conventional Electrocardiography (ECG). In this paper, we propose an IoT device utilizing a
                spintronic-technology-based ultra-sensitive Magnetic Tunnel Junction (MTJ) sensor that measures the
                magnetic fields produced by cardio-vascular electromagnetic activity, i.e. Magentocardiography (MCG). We
                treat the low-frequency noise generated by the sensor, which is also a challenge for most other sensors
                dealing with low-frequency bio-magnetic signals. Instead of relying on generic signal processing
                techniques such as moving average, we employ deep-learning training on bio-magnetic signals. Using an
                existing dataset of ECG records, MCG signals are synthesized. A unique deep learning model, composed of
                a one-dimensional convolution layer, Gated Recurrent Unit (GRU) layer, and a fully-connected neural
                layer, is trained using the labeled data moving through a striding window, which is able to smartly
                capture and eliminate the noise features. Simulation results are reported to evaluate the effectiveness
                of the proposed method that demonstrates encouraging performance.

              </p>
              <a href="https://doi.org/10.1109/ICC40277.2020.9148617"
                target="_blank"><small>https://doi.org/10.1109/ICC40277.2020.9148617</small></a>
              </p>
            </div>
          </li>

          <li>
            <a data-toggle="collapse" href="#pub4" class="collapsed">The impact of community containment implementation
              timing on the spread of COVID-19: A simulation study [version 1; peer review: 2 approved] <i
                class="icofont-simple-up"></i></a>
            <div id="pub4" class="collapse" data-parent=".faq-list">
              <p class="p-3">
              <h5>Authors</h5>
              <p>
                <strong>Attayeb Mohsen</strong>, Ahmed Alarabi
              </p>
              <h5>Journal</h5>
              <p>F1000Research vol. 9 452. 27 May. 2020</p>
              <h5>Abstract</h5>
              <div class="pr-3 pl-3">
                <h6>Background:</h6>
                <p>Community containment is one of the common methods used to mitigate infectious disease
                  outbreaks. The effectiveness of such a method depends on how strictly it is applied and the timing of
                  its
                  implementation. An early start and being strict is very effective; however, at the same time, it
                  impacts
                  freedom and economic opportunity. Here we created a simulation model to understand the effect of the
                  starting day of community containment on the ﬁnal outcome, that is, the number of those infected,
                  hospitalized and those that died, as we followed the dynamics of COVID-19 pandemic.</p>
                <h6>Methods: </h6>
                <p>We used a stochastic recursive simulation method to apply disease outbreak dynamics measures of
                  COVID-19 as an example to simulate disease spread. Parameters are allowed to be randomly assigned
                  between
                  higher and lower values obtained from published COVID-19 literature.</p>

                <h6>Results: </h6>
                <p>We simulated the dynamics of COVID-19 spread, calculated the number of active infections,
                  hospitalizations and deaths as the outcome of our simulation and compared these results with real
                  world
                  data. We also represented the details of the spread in a network graph structure, and shared the code
                  for
                  the simulation model to be used for examining other variables.</p>
                <h6>Conclusions: </h6>
                <p>Early implementation of community containment has a big impact on the ﬁnal outcome of an
                  outbreak.</p>


                </p>

              </div>
              <a href="https://doi.org/10.12688/f1000research.24156.1"
                target="_blank"><small>https://doi.org/10.12688/f1000research.24156.1</small></a>
          </li>
          <h4>2019</h4>
          <li>
            <a data-toggle="collapse" href="#pub5" class="collapsed">
              Impact of quality trimming on the efficiency of reads joining and diversity analysis of Illumina
              paired-end reads in the context of QIIME1 and QIIME2 microbiome analysis frameworks
              <i class="icofont-simple-up"></i></a>
            <div id="pub5" class="collapse" data-parent=".faq-list">
              <p class="p-3">
              <h5>Authors</h5>
              <p>
                <strong>Attayeb Mohsen</strong>, Jonguk Park, Yi-An Chen, Hitoshi Kawashima, Kenji Mizuguchi
              </p>
              <h5>Journal</h5>
              <p>
                BMC bioinformatics
              </p>
              <h5>Abstract</h5>
              <p class="pr-3 pl-3">
                To increase the accuracy of microbiome data analysis, solving the technical limitations of the existing
                sequencing machines is required. Quality trimming is suggested to reduce the effect of the progressive
                decrease in sequencing quality with the increased length of the sequenced library. In this study, we
                examined the effect of the trimming thresholds (0–20 for QIIME1 and 0–30 for QIIME2) on the number of
                reads that remained after the quality control and chimera removal (the good reads). We also examined the
                distance of the analysis results to the gold standard using simulated samples. Quality trimming
                increased the number of good reads and abundance measurement accuracy in Illumina paired-end reads of
                the V3-V4 hypervariable region. Our results suggest that the pre-analysis trimming step should be
                included before the application of QIIME1 or QIIME2.
              </p>
              <a href="https://doi.org/10.12688/f1000research.24156.1"
                target="_blank"><small>https://doi.org/10.12688/f1000research.24156.1</small></a>
              </p>
            </div>
          </li>
          <h4>2018</h4>
          <li>
            <a data-toggle="collapse" href="#pub6" class="collapsed">
              Effects of levocetirizine and diphenhydramine on regional glucose metabolic changes and hemodynamic
              responses in the human prefrontal cortex during cognitive tasks
              <i class="icofont-simple-up"></i></a>
            <div id="pub6" class="collapse" data-parent=".faq-list">
              <p class="p-3">
              <h5>Authors</h5>
              <p>
                Asuka Kikuchi, Fairuz Binti Mohammadi Nasir, Akie Inami, <strong>Attayeb Mohsen</strong>, Shoichi
                Watanuki, Masayasu
                Miyake, Kazuko Takeda, Daigo Koike, Takayasu Ito, Junpei Sasakawa, Rin Matsuda, Kotaro Hiraoka, Marcus
                Maurer, Kazuhiko Yanai, Hiroshi Watabe, Manabu Tashiro
              </p>
              <h5>Journal</h5>
              <p>
                Human Psychopharmacology: Clinical and Experimental/Volume 33, Issue 2/e2655
              </p>
              <h5>Abstract</h5>
              <div class="pr-3 pl-3">
                <h6>Objective</h6>
                <p>Antihistamines often have sedative side effects. This was the first study to measure regional
                  cerebral
                  glucose (energy) consumption and hemodynamic responses in young adults during cognitive tests after
                  antihistamine administration.</p>

                <h6>Methods</h6>
                <p>In this double‐blind, placebo‐controlled, three‐way crossover study, 18 healthy young Japanese men
                  received single doses of levocetirizine 5 mg and diphenhydramine 50 mg at intervals of at least six
                  days. Subjective feeling, task performances, and brain activity were evaluated during three cognitive
                  tests (word fluency, two‐back, and Stroop). Regional cerebral glucose consumption changes were
                  measured
                  using positron emission tomography with [18F]fluorodeoxyglucose. Regional hemodynamic responses were
                  measured using near‐infrared spectroscopy.</p>

                <h6>Results</h6>
                <p>Energy consumption in prefrontal regions was significantly increased after antihistamine
                  administration, especially diphenhydramine, whereas prefrontal hemodynamic responses, evaluated with
                  oxygenated hemoglobin levels, were significantly lower with diphenhydramine treatment. Stroop test
                  accuracy was significantly impaired by diphenhydramine, but not by levocetirizine. There was no
                  significant difference in subjective sleepiness.</p>

                <h6>Conclusions</h6>
                <p>Physiological “coupling” between metabolism and perfusion in the healthy human brain may not be
                  maintained under pharmacological influence due to antihistamines. This uncoupling may be caused by a
                  combination of increased energy demands in the prefrontal regions and suppression of vascular
                  permeability in brain capillaries after antihistamine treatment. Further research is needed to
                  validate
                  this hypothesis.</p>
              </div>
              <a href="https://doi.org/10.1002/hup.2655"
                target="_blank"><small>https://doi.org/10.1002/hup.2655</small></a>
              </p>
            </div>
          </li>
          <h4>2015</h4>
          <li>
            <a data-toggle="collapse" href="#pub7" class="collapsed">
              Histamine H3 receptor in primary mouse microglia inhibits chemotaxis, phagocytosis, and cytokine secretion
              <i class="icofont-simple-up"></i></a>
            <div id="pub7" class="collapse" data-parent=".faq-list">
              <p class="p-3">
              <h5>Authors</h5>
              <p>
                Tomomitsu Iida, Takeo Yoshikawa, Takuro Matsuzawa, Fumito Naganuma, Tadaho Nakamura, Yamato Miura,
                <strong>Attayeb S Mohsen</strong>, Ryuichi Harada, Ren Iwata, Kazuhiko Yanai
              </p>
              <h5>Journal</h5>
              <p>
                Glia, 63:7:123-1225
              </p>
              <h5>Abstract</h5>
              <p class="pr-3 pl-3">
                Histamine is a physiological amine which initiates a multitude of physiological responses by binding to
                four known G‐protein coupled histamine receptor subtypes as follows: histamine H1 receptor (H1R), H2R,
                H3R, and H4R. Brain histamine elicits neuronal excitation and regulates a variety of physiological
                processes such as learning and memory, sleep–awake cycle and appetite regulation. Microglia, the
                resident macrophages in the brain, express histamine receptors; however, the effects of histamine on
                critical microglial functions such as chemotaxis, phagocytosis, and cytokine secretion have not been
                examined in primary cells. We demonstrated that mouse primary microglia express H2R, H3R, histidine
                decarboxylase, a histamine synthase, and histamine N‐methyltransferase, a histamine metabolizing enzyme.
                Both forskolin‐induced cAMP accumulation and ATP‐induced intracellular Ca2+ transients were reduced by
                the H3R agonist imetit but not the H2R agonist amthamine. H3R activation on two ubiquitous second
                messenger signalling pathways suggests that H3R can regulate various microglial functions. In fact,
                histamine and imetit dose‐dependently inhibited microglial chemotaxis, phagocytosis, and
                lipopolysaccharide (LPS)‐induced cytokine production. Furthermore, we confirmed that microglia produced
                histamine in the presence of LPS, suggesting that H3R activation regulate microglial function by
                autocrine and/or paracrine signalling. In conclusion, we demonstrate the involvement of histamine in
                primary microglial functions, providing the novel insight into physiological roles of brain histamine.
              </p>
              <a href="https://doi.org/10.1002/glia.22812"
                target="_blank"><small>https://doi.org/10.1002/glia.22812</small></a>
              </p>
            </div>
          </li>

          <h4>2014</h4>
          <li>
            <a data-toggle="collapse" href="#pub8" class="collapsed">
              Insufficient intake of L-histidine reduces brain histamine and causes anxiety-like behaviors in male mice
              <i class="icofont-simple-up"></i></a>
            <div id="pub8" class="collapse" data-parent=".faq-list">
              <p class="p-3">
              <h5>Authors</h5>
              <p>
                Takeo Yoshikawa, Tadaho Nakamura, Tetsuro Shibakusa, Mayu Sugita, Fumito Naganuma, Tomomitsu Iida,
                Yamato Miura, <strong>Attayeb Mohsen</strong>, Ryuichi Harada, Kazuhiko Yanai
              </p>
              <h5>Journal</h5>
              <p>
                The Journal of nutrition:144:10:1637-1641
              </p>
              <h5>Abstract</h5>
              <p class="pr-3 pl-3">
                L-histidine is one of the essential amino acids for humans, and it plays a critical role as a component
                of proteins. L-histidine is also important as a precursor of histamine. Brain histamine is synthesized
                from L-histidine in the presence of histidine decarboxylase, which is expressed in histamine neurons. In
                the present study, we aimed to elucidate the importance of dietary L-histidine as a precursor of brain
                histamine and the histaminergic nervous system. C57BL/6J male mice at 8 wk of age were assigned to 2
                different diets for at least 2 wk: the control (Con) diet (5.08 g L-histidine/kg diet) or the low
                L-histidine diet (LHD) (1.28 g L-histidine/kg diet). We measured the histamine concentration in the
                brain areas of Con diet–fed mice (Con group) and LHD-fed mice (LHD group). The histamine concentration
                was significantly lower in the LHD group [Con group vs. LHD group: histamine in cortex (means ± SEs):
                13.9 ± 1.25 vs. 9.36 ± 0.549 ng/g tissue; P = 0.002]. Our in vivo microdialysis assays revealed that
                histamine release stimulated by high K+ from the hypothalamus in the LHD group was 60% of that in the
                Con group (P = 0.012). However, the concentrations of other monoamines and their metabolites were not
                changed by the LHD. The open-field tests showed that the LHD group spent a shorter amount of time in the
                central zone (87.6 ± 14.1 vs. 50.0 ± 6.03 s/10 min; P = 0.019), and the light/dark box tests
                demonstrated that the LHD group spent a shorter amount of time in the light box (198 ± 8.19 vs. 162 ±
                14.1 s/10 min; P = 0.048), suggesting that the LHD induced anxiety-like behaviors. However, locomotor
                activity, memory functions, and social interaction did not differ between the 2 groups. The results of
                the present study demonstrated that insufficient intake of histidine reduced the brain histamine
                content, leading to anxiety-like behaviors in the mice.
              </p>
              <a href="https://doi.org/10.3945/jn.114.196105"
                target="_blank"><small>https://doi.org/10.3945/jn.114.196105</small></a>
              </p>
            </div>
          </li>

          <li>
            <a data-toggle="collapse" href="#pub9" class="collapsed">
              Mechanism of the histamine H3 receptor-mediated increase in exploratory locomotor activity and
              anxiety-like behaviours in mice
              <i class="icofont-simple-up"></i></a>
            <div id="pub9" class="collapse" data-parent=".faq-list">
              <p class="p-3">
              <h5>Authors</h5>
              <p>
                <strong>Attayeb Mohsen</strong>, Takeo Yoshikawa, Yamato Miura, Tadaho Nakamura, Fumito Naganuma,
                Katsuhiko Shibuya, Tomomitsu Iida, Ryuichi Harada, Nobuyuki Okamura, Takehiko Watanabe, Kazuhiko Yanai
              </p>
              <h5>Journal</h5>
              <p>
                Neuropharmacology:81:188-194
              </p>
              <h5>Abstract</h5>
              <p class="pr-3 pl-3">
              <p>Histaminergic neurons are activated by histamine H3 receptor (H3R) antagonists, increasing histamine
                and other neurotransmitters in the brain. The prototype H3R antagonist thioperamide increases locomotor
                activity and anxiety-like behaviours; however, the mechanisms underlying these effects have not been
                fully elucidated. This study aimed to determine the mechanism underlying H3R-mediated behavioural
                changes using a specific H3R antagonist, JNJ-10181457 (JNJ).</p>

              <p>First, we examined the effect of JNJ injection to mice on the concentrations of brain monoamines and
                their metabolites. JNJ exclusively increased Nτ-methylhistamine, the metabolite of brain histamine used
                as an indicator of histamine release, suggesting that JNJ dominantly stimulates the release of histamine
                release but not of other monoamines.</p>

              <p>Next, we examined the mechanism underlying JNJ-induced behavioural changes using open-field tests and
                elevated zero maze tests. JNJ-induced increase in locomotor activity was inhibited by α-fluoromethyl
                histidine, an inhibitor of histamine synthesis, supporting that H3R exerted its effect through histamine
                neurotransmission. The JNJ-induced increase in locomotor activity in wild-type mice was preserved in H1R
                gene knockout mice but not in histamine H2 receptor (H2R) gene knockout mice. JNJ-induced anxiety-like
                behaviours were partially reduced by diphenhydramine, an H1R antagonist, and dominantly by zolantidine,
                an H2R antagonist. These results suggest that H3R blockade induces histamine release, activates H2R and
                elicits exploratory locomotor activity and anxiety-like behaviours.</p>
              </p>
              <a href="https://doi.org/10.1016/j.neuropharm.2014.02.003"
                target="_blank"><small>https://doi.org/10.1016/j.neuropharm.2014.02.003</small></a>
              </p>
            </div>
          </li>
        </ul>

      </div>
    </section><!-- End of Publications list Section -->

    <!-- ======= Contact Section ======= -->
    <section id="contact" class="contact">
      <div class="container" data-aos="fade-up">

        <div class="section-title">
          <h2>Contact</h2>
          <h3></h3>
          <p></p>
        </div>

        <div class="row" data-aos="fade-up" data-aos-delay="100">
          <div class="col-lg-6 col-md-6">
            <div class="info-box mb-4">
              <i class="bi bi-chat"></i>
              <h3>social </h3>
              <div class="social-links px-3" style="font-size: 2rem">
                <a href="https://www.linkedin.com/in/attayebmohsen/" class="px-3"><i
                    class="icofont-linkedin"></i></i></a>
                <a href="https://www.researchgate.net/profile/Attayeb-Mohsen" class="px-3"><i
                    class="ai ai-researchgate-square"></i></i></a>
                <a href="https://scholar.google.com/citations?user=SPmoxAoAAAAJ&hl=en" class="px-3"><i
                    class="ai ai-google-scholar-square"></i></i></a>
                <a href="https://orcid.org/0000-0003-0690-8012" class="px-3"><i class="ai ai-orcid-square"></i></i></a>
                <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mohsen+Attayeb" class="px-3"><i
                    class="ai ai-pubmed"></i></i></a>


              </div>
            </div>
          </div>

          <div class="col-lg-6 col-md-6">
            <div class="info-box  mb-4">
              <i class="bx bx-envelope"></i>
              <h3>Email Us</h3>
              <p>attayeb(@)nibiohn.go.jp</p>
            </div>
          </div>


        </div>

    </section><!-- End Contact Section -->

  </main><!-- End #main -->

  <!-- ======= Footer ======= -->
  <footer id="footer">

    <div class="footer-top">
      <div class="container">
        <div class="row">

          <div class="col-lg-4 col-md-6 footer-contact">
            <h3>Attayeb Mohsen<span>.</span></h3>
            <p>
              <br>

              <strong>Email:</strong> attayeb@nibiohn.go.jp<br>
            </p>
          </div>

          <div class="col-lg-4 col-md-6 footer-links">
            <h4>Useful Links</h4>
            <ul>
              <!-- 
              <li class="active"><a href="index.html">Home</a></li>
              <li><a href="#about">About</a></li>
              <li><a href="#skills">Skills</a></li>
              <li><a href="#publications">Publications</a></li>
              <li><a href="#contact">Contact</a></li>
              -->
              <li><i class="bx bx-chevron-right"></i> <a href="#">Home</a></li>
              <li><i class="bx bx-chevron-right"></i> <a href="#about">About us</a></li>
              <li><i class="bx bx-chevron-right"></i> <a href="#skills">Skills</a></li>
              <li><i class="bx bx-chevron-right"></i> <a href="#publications">Publications</a></li>
              <li><i class="bx bx-chevron-right"></i> <a href="#contact">Contact</a></li>
            </ul>
          </div>

          <div class="col-lg-4 col-md-6 footer-links">
            <h4>Our Social Networks</h4>
            <p></p>
            <div class="social-links mt-3">

              <a href="#" class="linkedin"><i class="bx bxl-linkedin"></i></a>
            </div>
          </div>

        </div>
      </div>
    </div>

    <div class="container py-4">
      <div class="copyright">
        &copy; Copyright <strong><span>Attayeb Mohsen</span></strong>. All Rights Reserved
      </div>
      <div class="credits">
        <!-- All the links in the footer should remain intact. -->
        <!-- You can delete the links only if you purchased the pro version. -->
        <!-- Licensing information: https://bootstrapmade.com/license/ -->
        <!-- Purchase the pro version with working PHP/AJAX contact form: https://bootstrapmade.com/bizland-bootstrap-business-template/ -->
        Designed by <a href="https://bootstrapmade.com/">BootstrapMade</a>
      </div>
    </div>
  </footer><!-- End Footer -->

  <div id="preloader"></div>
  <a href="#" class="back-to-top"><i class="icofont-simple-up"></i></a>

  <!-- Vendor JS Files -->
  <script src="assets/vendor/jquery/jquery.min.js"></script>
  <script src="assets/vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
  <script src="assets/vendor/jquery.easing/jquery.easing.min.js"></script>
  <script src="assets/vendor/php-email-form/validate.js"></script>
  <script src="assets/vendor/waypoints/jquery.waypoints.min.js"></script>
  <script src="assets/vendor/counterup/counterup.min.js"></script>
  <script src="assets/vendor/owl.carousel/owl.carousel.min.js"></script>
  <script src="assets/vendor/isotope-layout/isotope.pkgd.min.js"></script>
  <script src="assets/vendor/venobox/venobox.min.js"></script>
  <script src="assets/vendor/aos/aos.js"></script>

  <!-- Template Main JS File -->
  <script src="assets/js/main.js"></script>

</body>

</html>